include/openenclave/internal/sgx/plugin.h

```diff
@@ -3,7 +3,7 @@
 
 #ifndef _OE_INTERNAL_SGX_PLUGIN
 #define _OE_INTERNAL_SGX_PLUGIN
-
+#include <openenclave/bits/evidence.h>
```

Comment:
Wonder why this header file should be included?

Reply:
removed